### PR TITLE
Release 2.24

### PR DESCRIPTION
**Pull request by larswmh**
This PR contains all changes to the website required for version 2.24. They are listed in this issue: #2924

### CODE REVIEW

**Comment by MikeMcC399** [comment]
Manually executing `npm test` passes on this branch 👍🏻

File: src/data/faq.json [hunk 1]
```diff
@@ -463,24 +463,30 @@
                                 ]
                             },
                             {
-                                "title": "How do I create a printout of my EU Digital COVID Certificate?",
+                                "title": "How do I create a printout of EU Digital COVID Certificates?",
                                 "anchor": "eu_dcc_export",
                                 "active": false,
                                 "textblock": [
+                                    "<b>Updated ??? ?? ????</b><hr>",
```
Comment (MikeMcC399) [inline src/data/faq.json:470]: This line needs to be updated before merging.

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Perhaps this PR should be in draft status or https://github.com/corona-warn-app/cwa-website/labels/do-not-merge until it is ready?

**Comment by larswmh** [comment]
Any that is missing by now, such as the release date, will be added shortly before merging

**Comment by MikeMcC399** [comment]
http://localhost:8000/assets/screenshots/2-24/en/android/VaccinationDetailsFragment_immune_2.png is incorrect.

image

The text "Please make sure to get a new digital certificate in time." has been removed from the app.

- See https://github.com/corona-warn-app/cwa-app-android/pull/5282

**Comment by MikeMcC399** [comment]
@larswmh 

Where the text  "Please make sure to get a new digital certificate in time."  is removed in 2.24, it does not leave a blank space, instead the text "Technical Expiration Date<br>Valid until mm/dd/yy, hh:mm AM" becomes visible.

image

**Comment by larswmh** [comment]
@MikeMcC399 

> Where the text "Please make sure to get a new digital certificate in time." is removed in 2.24, it does not leave a blank space, instead the text "Technical Expiration Date
Valid until mm/dd/yy, hh:mm AM" becomes visible.

Thanks for your comment. In addition to fixing this, I have replaced both VaccinationDetailsFragment, which were incorrectly invalid, with a valid version.

**Comment by MikeMcC399** [comment]
@larswmh 

The blog has been published under the link
- https://www.coronawarn.app/en/blog/2022-06-29-cwa-2-24/ and
- https://www.coronawarn.app/de/blog/2022-06-29-cwa-2-24/

however the FAQ article (EN)
- https://www.coronawarn.app/en/faq/results/#eu_dcc_export and
links to the non-existent 
https://www.coronawarn.app/en/blog/2022-06-29-cwa-version-2-24/

and the FAQ article (DE)
- https://www.coronawarn.app/de/faq/results/#eu_dcc_export and
links to the non-existent 
https://www.coronawarn.app/de/blog/2022-06-29-cwa-version-2-24/

The links to the blog need to be corrected. Will you do this?

**Pull request merged**